cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/04/s3_dataset.spec.js

```diff
@@ -100,6 +100,10 @@ const s3DatasetTestSuite = () => {
         });
         afterEach(() => {
           cy.deleteWorkspaceByName(workspace);
+          cy.window().then((win) => {
+            win.localStorage.clear();
+            win.sessionStorage.clear();
+          });
```

Comment:
```suggestion
          cy.clearLocalStorage();
          cy.clearAllSessionStorage();
```

are these not available? source: https://docs.cypress.io/api/commands/clearlocalstorage

also should we consider the clearing out in the `beforeEach`? in case a different test file doesn't follow best practices and clears out this info. so ensuring that this test run is clean. regardless if the previous test cleared out local storage and session storage or not

Reply:
Was following other tests to see how they were doing it: https://github.com/search?q=repo%3Aopensearch-project%2FOpenSearch-Dashboards%20cy.window()&type=code

Reply:
> Other tests' caches shouldn't affect this one.

True as of today, but if another dev decides to add another test file that touches the same stuff this spec tests but dosn't clear it then this test could be potentially impacted